core/src/wallet/bitcoin/BitcoinLikeAccount.cpp

```diff
@@ -630,7 +630,7 @@ namespace ledger {
         std::shared_ptr<api::BitcoinLikeTransactionBuilder> BitcoinLikeAccount::buildTransaction(std::experimental::optional<bool> partial) {
             auto self = std::dynamic_pointer_cast<BitcoinLikeAccount>(shared_from_this());
             auto getUTXO = [=]() -> Future<std::vector<BitcoinLikeUtxo>> {
-                return async<std::vector<BitcoinLikeUtxo>>([=]() {
+                return Future<std::vector<BitcoinLikeUtxo>>::async(_executionContext, [=]() {
```

Comment:
Can you explain me what I was doing wrong here ?

Reply:
Nothing wrong, just MSVC can't resolve between `async(...)` and `async(...) const` versions of the function.

Reply:
Ok ! What about the `_executionContext` you are using in your change ?

Reply:
I just "inline" `async` call. `BitcoinLikeAccount` is derived from `DedicatedContext` that contains `async` method and `_executionContext` member, so no semantic changes here.